.travis.yml

```diff
@@ -55,4 +55,9 @@ before_script:
 script:
   - java -version
   - bin/scalafmt --version 0.4.7 --test
+  - git clone https://github.com/MasseGuillaume/sbt-cross-project.git
+  - pushd sbt-cross-project
+  - git checkout simple2
+  - sbt publishLocal
+  - popd
```

Comment:
What's the correct merge order between this pr and scala-native/sbt-cross-project#4 ?